File: front/src/enum/SingerEnum.ts
export enum SingerEnum {
    FEMALE_SINGER = 1,
    MALE_SINGER = 2,
    GROUP = 3,
    ALL_SINGER = 0,
}


export const singer_style_list = [
    {
        id: SingerEnum.ALL_SINGER,
        name: '全部歌手'
    },
    {
        id: SingerEnum.FEMALE_SINGER,
        name: '女歌手'
    },
    {
        id: SingerEnum.MALE_SINGER,
        name: '男歌手'
    },
    {
        id: SingerEnum.GROUP,
        name: '组和'
    },
]

File: src/main/java/com/wll/enums/SingerEnum.java
package com.wll.enums;

/**
 * @time 2025/3/11 23:42 周二
 */
public enum SingerEnum {
    ALL_SINGER(0),

    FEMALE_SINGER(1),

    MALE_SINGER(2),

    GROUP(3);

    private final int id;
    private final String value;

    SingerEnum(int id) {
        this.id = id;
        value = "singer:" + id;
    }

    public int getId() {
        return id;
    }


    public String getValue() {
        return value;
    }
}
